app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListFragment.java

```diff
@@ -37,6 +40,8 @@
         NameEditChoicesDialog.Listener, RenameDialog.Listener, DeleteNameDialog.Listener,
         DuplicationDialog.Listener, MergeNameListsDialog.Listener {
 
+    private static final int SPEECH_REQUEST_CODE = 4;
```

Comment:
nit: We can just use 1 here.

Reply:
You avoid defining it as public static in SpeechUtil file to prevent some kind of hackers access it or from other files?